sisl/viz/plotly/plots/grid.py

```diff
@@ -609,6 +618,34 @@ def _transform_grid(grid, transform):
                 transform = f"numpy.{transform}"
 
         return grid.apply(transform)
+    
+    @staticmethod
+    def _get_representation(grid, represent):
+        """Returns a representation of the grid
+        
+        Parameters
+        ------------
+        grid: sisl.Grid
+            the grid for which we want return
+        represent: {"real", "imag", "mod", "deg_phase", "rad_phase"}
```

Comment:
perhaps a default `phase` which is using `radians`?

Reply:
Actually, currently anything that ends with `"phase"` and doesn't begin with `"deg"` is interpreted as `"rad_phase"`. I should probably be more strict and accept only "phase", "rad_phase" and "deg_phase", right? :sweat_smile:

Reply:
Yeah ;)